ra/ra.go

```diff
@@ -993,6 +1010,37 @@ func (ra *RegistrationAuthorityImpl) issueCertificate(
 			// This context is limited by the gRPC timeout.
 			_ = ra.publisher.SubmitToCT(context.Background(), cert.DER)
 		}()
+	} else if ra.ctpolicy != nil {
```

Comment:
It looks like this will only happen if publisher is nil. But AFAICT above, publisher will always be non-nil. So probably we should swap this if clause with the one above it.